cve_bin_tool/version.py

```diff
@@ -10,6 +10,10 @@
 
 VERSION: str = "3.2.2dev0"
 
+HTTP_HEADERS: dict = {
```

Comment:
My only comment here is that this is such a good idea it may be worth adding an environment variable, command line arg, or configuration option to let the user optionally specify this value. The list of sources doesn't have anything I would anticipate adding user-agent checks, but it's possible corporate firewalls or some such thing may restrict outbound user agents.